.gitignore

```diff
@@ -25,6 +25,7 @@ coverage
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+yarn.lock
```

Comment:
yarn.lock should always be checked in to ensure that the dependencies are predictably installed